lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -132,20 +132,12 @@ spec = do
         prop "roundtrip for Shelley witnesses" prop_decodeSignedShelleyTxRoundtrip
         prop "roundtrip for Byron witnesses" prop_decodeSignedByronTxRoundtrip
 
-    estimateMaxInputsTests @ShelleyKey Cardano.Mainnet
-        [(1,23),(10,16),(20,9),(30,2)]
-    estimateMaxInputsTests @ShelleyKey (Cardano.Testnet (Cardano.NetworkMagic 0))
-        [(1,23),(10,16),(20,9),(30,2)]
-
-    estimateMaxInputsTests @ByronKey Cardano.Mainnet
-        [(1,16),(10,10),(20,5),(30,0)]
-    estimateMaxInputsTests @ByronKey (Cardano.Testnet (Cardano.NetworkMagic 0))
-        [(1,15),(10,10),(20,4),(30,0)]
-
-    estimateMaxInputsTests @IcarusKey Cardano.Mainnet
-        [(1,19),(10,14),(20,9),(30,4)]
-    estimateMaxInputsTests @IcarusKey (Cardano.Testnet (Cardano.NetworkMagic 0))
-        [(1,18),(10,13),(20,8),(30,2)]
+    estimateMaxInputsTests @ShelleyKey
+        [(1,27),(10,19),(20,10),(30,1)]
+    estimateMaxInputsTests @ByronKey
+        [(1,17),(10,11),(20,4),(30,0)]
+    estimateMaxInputsTests @IcarusKey
+        [(1,17),(10,11),(20,4),(30,0)]
```

Comment:
NOTE 1: IcarusKey and ByronKey are now strictly equal since the estimation function doesn't make a difference between both. 

NOTE 2: Similarly, there's no longer a variation between mainnet and testnet (this only affected Byron addresses, and this new approach just consider the worst case scenario for Byron addresses all the time).